### PR TITLE
Using Vue.js to the player page

### PR DESCRIPTION
**Pull request by philpichet**

### CODE REVIEW

File: player.html [hunk 1]
```diff
@@ -93,90 +94,89 @@
...
-        </ul>
-        <div id="bank" data-bind="text: bankAmountThisRound"></div>
+  </style>
+</head>
+<body>
+  <div id="game_container">
+    <div id="game">
+      <div v-show="notConnected" id="nohost">NO HOST CONNECTED</div>
+      <ul id="ladder">
+        <li v-for="(step, index) in steps.slice().reverse()" v-bind:class="isCurrentAmount(step)">{{ step }}</li>
```
Comment (Neamar) [inline player.html]: Do you need the index?

File: player.html [hunk 2]
```diff
@@ -93,90 +94,89 @@
...
-    state.reversedSteps = ko.computed(function() {
-      return this.steps().slice(0).reverse();
-    }, state);
-    state.valueStatus = function(index) {
-      return (this.steps().length - this.currentStep() - 1) <= index() ? "active" : "";
-    };
-
-    // Activates knockout.js
-    ko.applyBindings(state, document.body);
+    // Instanciate Vue
```
Comment (Neamar) [inline player.html]: Instantiate

File: player.html [hunk 3]
```diff
@@ -93,90 +94,89 @@
...
+      el: '#game_container',
+      data: {
+        notConnected: true, 
+        steps: [],
+        bankAmountThisRound: 0,
+        timeLeft: 0,
+        audioBlocked: true
+      },
+      methods: {
+        isCurrentAmount(step) {
```
Comment (Neamar) [inline player.html]: Functions starting with 'is' sound like they'll return a boolean.

Maybe getClassForAmount instead?

File: player.html [hunk 4]
```diff
@@ -93,90 +94,89 @@
...
+          return this.currentAmount >= step ? 'active' : ''
+        }
+      }
+    });
 
-    // Test if webaudio is active and if not, wait for first user input to start it
+     // Test if webaudio is active and if not, wait for first user input to start it
```
Comment (Neamar) [inline player.html]: Indent seems off

File: player.html [hunk 5]
```diff
@@ -93,90 +94,89 @@
...
 
-        const newState = JSON.parse(newStateString);
-        // Mirror changes on main object
-        ['steps', 'bankAmountThisRound', 'currentAmount', 'timeLeft', 'currentPlayer', 'currentStep'].forEach(k => {
-          state[k](newState[k]);
-        });
+      const newState = JSON.parse(newStateString);
+      // Mirror changes on main object
+      ['steps', 'bankAmountThisRound', 'currentAmount', 'timeLeft', 'currentPlayer', 'currentStep'].forEach(k => {
+        console.log(newState[k])
```
Comment (Neamar) [inline player.html]: Console logging

File: player.html [hunk 6]
```diff
@@ -93,90 +94,89 @@
...
+        currentAudio.play();
+      }
 
-        if(!newState.timerIsRunning) {
-          state.timeLeft(newState.roundDuration);
-        }
-      });
-    </script>
-  </body>
+      // Reset timer in the end of tour
```
Comment (Neamar) [inline player.html]: Tour => turn

File: player.html (hunk 1 above)
Comment (philpichet) [reply player.html]: The first time, i trie yes, Not now.

File: player.html (hunk 6 above)
Comment (philpichet) [reply player.html]: Not for google ;)

**Pull request merged**